Week-1/index.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Akshay</title>
+    <link rel="stylesheet"
+        href="https://maxst.icons8.com/vue-static/landings/line-awesome/font-awesome-line-awesome/css/all.min.css">
```

Comment:
@relhanakshay you are loading font-awesome css to use their icons right. Why do we need to have those images as part of source control ? Can we use it from that ?

Reply:
@relhanakshay we can refer to the image classnames and use it from font-awesome. Please remove those images from your commit

Reply:
Hi Rajesh, I am actually using images from the folder. I will remove this CSS link.

Reply:
ok. As per instructions we can use font-awesome icons. Any issues ?